yt_dlp/extractor/netverse.py

```diff
@@ -0,0 +1,174 @@
+            'id': 'x8278vk',
+            'ext': 'mp4',
+            'access_id': 'k4CNGz7V0HJ7vfwZbXy',
+            'title': 'Tetangga Baru',
+            'season': 'Season 1',
+            'description': 'md5:ed6dd355bed84d139b1154c3d8d65957',
+            'thumbnail': 'https://vplayed-uat.s3-ap-southeast-1.amazonaws.com/images/webseries/thumbnails/2021/11/619cfd9d32c5f.jpeg',
+            'episode_number': 1,
+            'series': 'Tetangga Masa Gitu',
+            'episode': 'Episode 1',
+        }}, { # noqa
+        # /video url
+        'url': 'https://www.netverse.id/video/pg067482-hellojadoo-season1',
+        'title': 'Namaku Choi Jadoo',
+        'info_dict': {
+            'id': 'x887jzz',
+            'ext': 'mp4',
+            } # noqa
+        }]  # noqa
+
+    def _real_extract(self, url):
+        program_json = self.get_required_json(url=url)
+
+        videos = traverse_obj(program_json, ('response', 'videos'))
+
+        video_url = videos.get('dailymotion_url')
+        episode_order = videos.get('episode_order')
+
+        program_detail = videos.get('program_detail')
+
+        # actually the video itself in daily motion, but in private
+        # Maybe need to refactor
+        access_id, real_video_json = self._call_metadata_api_from_video_url(video_url)
+
+        video_id = real_video_json.get('id')
+
+        # For m3u8
+        m3u8_file = traverse_obj(real_video_json, ('qualities', 'auto'))
+
+        for format in m3u8_file:
+            video_url = format.get('url')
+            if video_url is None:
+                continue
+            self.video_format = self._extract_m3u8_formats(video_url, video_id=video_id)
+# noqa W293
+        episode = f'Episode {episode_order}'
+
+        self._sort_formats(self.video_format)
+        return {
+            'id': video_id,
+            'access_id': access_id,
+            'formats': self.video_format,
+            'title': videos.get('title'),
+            'season': videos.get('season_name'),
+            'thumbnail': program_detail.get('thumbnail_image'),
+            'description': program_detail.get('description'),
+            'episode_number': videos.get('episode_order'),
+            'series': program_detail.get('title'),
+            'episode': episode,
+
+        }
+
+
+class NetversePlaylistIE(NetverseBaseIE):
+    _VALID_URL = r'https?://(?:\w+\.)?netverse\.id/(?P<type>webseries)/(?P<display_id>[^/?#&]+)'
+    _TEST = {
+        'url': 'https://netverse.id/webseries/tetangga-masa-gitu',
+        'info_dict': {
+            'id': 16,
+            'title': 'Tetangga Masa Gitu',
+        },
+        'playlist_mincount': 10,
+        'params': {
+            'skip_download': True,
+        }
+    }
+
+    def _real_extract(self, url):
+        playlist_data = self.get_required_json(url)
+
+        webseries_info = traverse_obj(playlist_data, ('response', 'webseries_info'))
+
+        videos = traverse_obj(playlist_data, ('response', 'related', 'data'))
```

Comment:
```suggestion
        playlist_data = self.get_required_json(url)
        webseries_info = traverse_obj(playlist_data, ('response', 'webseries_info'))
        videos = traverse_obj(playlist_data, ('response', 'related', 'data'))
```
why is there space after each line!?

(Same above)